packages/slate-editor/src/modules/popper-options-context/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { PreventOverflowModifier } from '@popperjs/core/lib/modifiers/preventOverflow';
+
+export interface Context {
+    modifiers?: {
+        preventOverflow?: PreventOverflowModifier['options'];
+    };
+    placement?: 'left-start' | 'right-start';
```

Comment:
Is it possible to make it automatic, if there is not place on the right move it to the left, probably there should be such props for popper

Reply:
Yes, it is possible but it doesn't work well in the story editor because when the toolbox goes to the left, it's covered by the sidebar due to z-index issues there.

It would require more work to make this work nicely so I've opted for this simpler approach, but it's definitely doable.

Reply:
I have some suggestion to make the menu adaptive. Feel free to apply these changes in some way or just cancel that PR https://github.com/prezly/slate/pull/338